engine.jackson/src/test/java/org/opencds/cqf/cql/engine/execution/CqlErrorsAndMessagingOperatorsTest.java

```diff
@@ -1,18 +1,18 @@
 package org.opencds.cqf.cql.engine.execution;
 
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.is;
-
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.function.Supplier;
-
 import org.opencds.cqf.cql.engine.data.SystemDataProvider;
 import org.opencds.cqf.cql.engine.elm.execution.obfuscate.PHIObfuscator;
 import org.opencds.cqf.cql.engine.elm.execution.obfuscate.RedactingPHIObfuscator;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.function.Supplier;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
```

Comment:
Lots of import reordering here that appears to be the reverse of what's typically being used. What Java formatter are you using?

Reply:
IntelliJ :( I need to activate it for some projects and deactivate it for others. I will revert these imports.